### ISSUE TITLE
Excuse me, I deployed LLaVA-med locally, why is my answer only one word?

### ISSUE THREAD

**Issue by ZG-yuan** [microsoft/LLaVA-Med#86]
Some weights of the model checkpoint at /mnt/disk/zgy/LLaVA-Med/LLaVA-Med-main/llava-med-v1.5-mistral-7b were not used when initializing LlavaMistralForCausalLM: ['model.vision_tower.vision_tower.vision_model.encoder.layers.19.mlp.fc1.weight',..........]
USER: is heart CT?
The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Setting `pad_token_id` to `eos_token_id`:2 for open-end generation.
ASSISTANT: Yes
USER: is leg CT?
The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Setting `pad_token_id` to `eos_token_id`:2 for open-end generation.
ASSISTANT: No
USER:What is it?
The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Setting `pad_token_id` to `eos_token_id`:2 for open-end generation.
ASSISTANT: This

**Comment by Rickylht** [comment]
Same bug here. Still debugging.
I think they change the end_of_text token when do training.

**Comment by Rickylht** [comment]
You need to change the template in conversation.py 
![image](https://github.com/user-attachments/assets/cf81bf4d-5174-4b15-9639-dbf08ccbef87)
replace the `sep=""` as `sep="<s>".` 
Otherwise, the mode will regard blank space as stop token. 
Then everything goes well. 
![image](https://github.com/user-attachments/assets/b2983c33-152c-4b64-8efd-f86528fb8f3d)
Strange bug solved 🕶️

**Comment by ZG-yuan** [comment]
@Rickylht Thank you very much!
